cylc/uiserver/handlers.py

```diff
@@ -246,9 +256,13 @@ def set_default_headers(self) -> None:
     @web.authenticated
     # @authorised  TODO: I can't think why we would want to authorise this
     def get(self):
-        user_info = {
-            'name': get_username(self)
-        }
+        user_info = dict(self.current_user.__dict__)
+
+        user_details = get_usernames(self)
+
+        user_info['name'] = user_details['name']
+        user_info['initials'] = user_details['initials']
+        user_info['username'] = user_details['username']
```

Comment:
To combine the fields in these two dictsm can do something like:

```python
user_info = {
    **self.current_user.__dict__,
    **get_usernames(self)
}
```

(this "unpacks" the provided dictionaries into a new one)